amethyst_assets/src/prefab/mod.rs

```diff
@@ -246,7 +246,9 @@ impl<T> Prefab<T> {
     ///
     /// If sub asset loading have not been triggered
     pub fn progress(&self) -> &ProgressCounter {
-        self.counter.as_ref().unwrap()
+        self.counter
+            .as_ref()
+            .expect("Sub asset loading have not been triggered")
```

Comment:
has*
The mistake was there before in the doc in fact, could you fix it in the process?

Reply:
Thank you for fast feedback @Moxinilian! I sorry about all the mishaps, corrections incoming 👍